.storybook/webpack.config.js

```diff
@@ -2,17 +2,17 @@ const path = require('path')
 const genDefaultConfig = require('@storybook/react/dist/server/config/defaults/webpack.config.js')
 
 module.exports = (config, env) => {
-    const myConfig = genDefaultConfig(config, env)
+  const myConfig = genDefaultConfig(config, env)
```

Comment:
we have `.vscode` in .gitignore, so just drop those new settings in your `settings.json`... which i'm sure you know exactly what i'm talking about, so i'll shutup now.